packages/ui/src/components/shared/export/Json.tsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+import { AccountBackup } from '@coong/keyring/types';
+import { Download } from '@mui/icons-material';
+import { Button } from '@mui/material';
+import FileSaver from 'file-saver';
+import { ExportObject, Props } from 'types';
+
+interface JsonProps extends Props {
+  value: AccountBackup;
+  object: ExportObject;
+  detail?: string;
+}
+
+export default function Json({ value, object, detail = '' }: JsonProps): JSX.Element {
```

Comment:
I think `JsonFile` might be more clear than just `Json` here.
```suggestion
export default function JsonFile({ value, object, detail = '' }: JsonProps): JSX.Element {
```